Example/Example.xcodeproj/project.pbxproj

```diff
@@ -367,20 +367,21 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-
-				DEVELOPMENT_TEAM = C65W8JU69P;
-
+				CODE_SIGN_IDENTITY = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 4R67Q8JL33;
```

Comment:
Could you explain the difference between both `DEVELOPMENT_TEAM` ?
Why do we need both and one is empty ?

Reply:
This auto-generated file is from IDK, It is responsible for maintaining references to all of the linked files and their groupings, linked frameworks, and, most importantly, the project's build settings, so for this commit yesterday I tried to archive the demo app for that we need a development team

Reply:
OK understood.